src/reportError.ts

```diff
@@ -69,3 +72,12 @@ function printRelatedSources (msg: Object) {
 function printErrorSummary (message: string) {
   console.log(chalk.red('ERROR'), message)
 }
+
+function reportModifiedDependency (err: Error, msg: Object) {
+  printErrorSummary('Packages in the store have been mutated')
+  console.log()
+  console.log('These packages are modified:')
+  msg['modified'].forEach((pkgPath: string) => console.log(chalk.gray(pkgPath)))
+  console.log()
```

Comment:
there is somewhat difference `console.log` is blocking sync operation, so if you can reduce amount of it, it might help with speed